docs/sources/next/javascript-api/k6-experimental/browser/request/timing.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'timing()'
+description: 'Browser module: Request.timing method'
+---
+
+# timing()
+
+Returns resource timing information for given request. Most of the timing values become available upon the response, `responseEnd` becomes available when request finishes.
+
+### Returns
+
+| Type                                 | Description                                                                                                                                                                                                                                 |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ResourceTiming                       | The object that is returned.                                                                                                                                                                                                                |
+| ResourceTiming.statTime              | Request start time in milliseconds elapsed since January 1, 1970 00:00:00 UTC                                                                                                                                                               |
```

Comment:
This also looks fine. However, we did this differently in `Frame.boundingBox` ([here](https://github.com/grafana/k6-docs/blob/main-browser-async/docs/sources/next/javascript-api/k6-experimental/browser/elementhandle/boundingbox.md)) after having a discussion. So, no strong opinion. But, should we follow the same approach for consistency?

```
### Returns

| Type                                 | Description                                                                                                                                                                                                                                 |
| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [ResourceTiming](#ResourceTiming)    | The object that is returned.                                                                                                                                                                                                                |

### ResourceTiming

...
```

```

Reply:
Updated in 26b0b1b75de51af3be46b71a90cfd79f7966d14a